resources/views/components/form/button.blade.php

```diff
@@ -1,3 +1,5 @@
 <div class="form-group text-center">
-    {{ Form::button($value, array_merge_concat(['class' => 'btn btn-default'], $options)) }}
-</div>
\ No newline at end of file
+    <button type="button" class="btn btn-default {{ $options['class'] ?? '' }}" @if(isset($options['id'])) id="{{ $options['id'] }}" @endif @if(isset($options['name'])) name="{{ $options['name'] }}" @endif>
```

Comment:
Does this need `type="submit"' if its a form button?

Reply:
Nope, that would be the submit.blade.php - Form::submit vs Form::button were different in the Collective package.